examples/gatsby/plugins/gatsby-plugin-material-ui/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "gatsby-source-materal-ui"
```

Comment:
should it be `gatsby-plugin-mui-emotion`? at least not with `gatsby-source-*`

https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/naming-a-plugin/

Reply:
Didn't payed too much attention on the name, will rename it :)

Reply:
```suggestion
  "name": "gatsby-plugin-mui-emotion"
```